server/l10n/messages.pot

```diff
@@ -207,6 +190,41 @@ msgstr ""
 msgid "Gross National Income"
 msgstr ""
 
+#. Blog post entry. Please take care to maintain valid <a> tags in translation. See this string in action at https://datacommons.org
+#: server/templates/static/homepage.html:56
+msgid "HOME-BLOG_POST_1"
+msgstr "%(project)s is now accessible on Google Search! <a href=%(blog_url)s>Read more here</a>"
+
+#. Describes building your own tools and analysis using our API (here Python refers to the programming language - see python.org, and REST is an acronym for REST is an acronym for REpresentational State Transfer). Please see this string in action at https://datacommons.org
+#: server/templates/static/homepage.html:143
+msgid "HOME-BUILD_CALLOUT"
+msgstr "Use the Python and REST API's to do your own custom analysis, build your own tools and more."
+
+#. A header of a section showing the many data sources included in the Data Commons Graph. See this string at https://datacommons.org
+#: server/templates/static/homepage.html:176
```

Comment:
curious if the line content changes, does it matter?

Reply:
It doesn't matter - this file is automatically generated. I've found it helpful to see the pointer to the source of the string though.